packages/components/src/components/data-list/data-list.tsx

```diff
@@ -102,7 +102,19 @@ const DataList = React.memo(
 
         const rowRenderer = ({ style, index, key, parent }: ListRowProps) => {
             const { getRowAction, passthrough, row_gap } = other_props;
-            const row = data_source[index];
+            let row = data_source[index];
+            if (
+                isForwardStartingBuyTransaction(
+                    row.action_type,
+                    row.shortcode,
+                    row.purchase_time || row.transaction_time
```

Comment:
Can we pass a single `row` param and destructure/use the properties we need inside isForwardStartingBuyTransaction? :)
same in data-table.tsx

Reply:
But row object has a lot of fields, a we need only 3. What are the benefits of passing all object? Moreover, in that case it won't be obvious which exactly information do we need